cms/tests/test_migrations.py

```diff
@@ -0,0 +1,38 @@
+from django.core.management import call_command
+from django.test import TestCase, override_settings
+from django.utils.six import text_type
+from django.utils.six.moves import StringIO
+
+from cms.utils.compat import DJANGO_1_9
+
+class MigrationTestCase(TestCase):
+
+    @override_settings(MIGRATION_MODULES={})
+    def test_for_missing_migrations(self):
+        output = StringIO()
+        options = {
+            'interactive': False,
+            'dry_run': True,
+            'stdout': output,
+        }
+
+        if DJANGO_1_9:
+            # this was deprecated on Django 1.10
+            options['exit_code'] = True
+        else:
+            options['check_changes'] = True
```

Comment:
You could put this in the original dict and delete the key in the DJANGO_1_9 block so you can just remove that when dropping support for old versions.